### PR TITLE
README.MD file updated

### PR DESCRIPTION
**Pull request by aakashrana007**
I've updated README.MD file and which is yet to be updated but since I have made a layout, it'd be easier later.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,74 @@
...
+3. Visit the website at _http://localhost:4000_.
+
+### Contributing
+
+Please refer to the [contributing.md](CONTRIBUTING.MD) file for detailed instructions on how to contribute to this project. It outlines the workflow for making changes, submitting pull requests, and reviewing contributions.
+
+### File Conventions
+To maintain code consistency and best practices, we follow specific conventions for structuring files and code. Please refer to the file-convention.md document for more information.
+
+### License
```
Comment (gaurovgiri) [inline README.md]: No license is required for now.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,74 @@
...
+
+`git clone https://github.com/kec-computer-club/website.git`
+   
+2. Navigate to the project directory:
+
+`cd path_to_your_folder`
+
+3. Build and run the Docker container:
+
+`docker-compose up`
```
Comment (gaurovgiri) [inline README.md]:
```suggestion
`docker-compose up` or `docker compose up`
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,74 @@
...
+### Contributing
+
+Please refer to the [contributing.md](CONTRIBUTING.MD) file for detailed instructions on how to contribute to this project. It outlines the workflow for making changes, submitting pull requests, and reviewing contributions.
+
+### File Conventions
+To maintain code consistency and best practices, we follow specific conventions for structuring files and code. Please refer to the file-convention.md document for more information.
+
+### License
+This project is licensed under the MIT License. See the LICENSE file for details.
+
```
Comment (gaurovgiri) [inline README.md]: Display the contributors.

**Review by gaurovgiri — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,2 +1,71 @@
...
+## Getting Started
+
+### Prerequisites
+
+To work on this project, ensure you have the following installed:
+
+- [Docker](https://www.docker.com/) (for running the site locally in a container)
+- [Jekyll](https://jekyllrb.com/) (for building and serving the site)
+- [Git](https://git-scm.com/) (for version control)
+
```
Comment (gaurovgiri) [inline README.md:29]: add tailwind as well

**Review by gaurovgiri — CHANGES_REQUESTED** [review]

**Review by gaurovgiri — APPROVED** [review]
LGTM!

**Review by KhatiwadaPrajwal056 — APPROVED** [review]
LGTM

**Review by prabhavpokharel — APPROVED** [review]
LGTM

**Pull request merged**